ncurses.yaml

```diff
@@ -1,7 +1,7 @@
 package:
   name: ncurses
   version: 6.4
```

Comment:
Maybe we should do something like

```suggestion
  version: 6.4_p20230701
```

Reply:
updated, still can't auto update with wolfictl, but at least it's clearer now when we're bumping the source

does this need a provides `ncurses=6.4`? or is it smart enough to figure that out